config/routes.rb

```diff
@@ -16,14 +16,38 @@
       defaults: { format: :json }
   end
 
-  resources :app_groups, 
-    only: %i[index show new create], 
+  get '/users/search', to: 'users#search', defaults: { format: :json }
+  get '/groups/search', to: 'groups#search', defaults: { format: :json }
+  match '/app_group_users/:user_id/set_role', to: 'app_group_users#set_role', via: [:put, :patch], as: 'set_role_app_group_user'
+  delete '/app_group_users/:user_id/delete/:app_group_id', to: 'app_group_users#destroy', as: 'app_group_user'
+
+  resources :app_group_admins,
```

Comment:
do we still need `app_group_admins` & `app_group_permissions` routes? please also check for other unused routes